### ISSUE TITLE
cant read property 'bind of undefined

### ISSUE THREAD

**Issue by devmondo** [mojo-js/paperclip.js#228]
sorry to post again so fast, but i am really not sleeping today exploring this :)

i use es6 and jspm loader and system.js  it is the future of working
now i can load html template with systemjs text plugin and compile  just fine but as soon as the html template contains input fields i get the below error

here is my js code
```
import paperclip from 'paperclip'
import tpl from './template.html!text'
export default class Test {
    constructor(){
        console.log(tpl)
        // create the template. accepts a string, or pre-compiled template
        this.template = paperclip.template(tpl);
        //this.numItems= 1000;

// create the view from the template
        this.view = this.template.view({
            numItems: 1000,
            range: function (count) {
                return _.range(Math.min(count, 20000)).reverse();
            }
        });
// render the view, and append to the DOM
// Should say "hello world!"
        document.body.appendChild(this.view.render());

    }
 
}
new Test();
```
```
///this is console log of the html template so i can make sure html is loaded, just testing
<repeat each="{{ range(numItems) }}" as='number'>
    {{~number}} <br />
</repeat>

es6-module-loader.src.js:140 Potentially unhandled rejection [2] TypeError: Error loading "main" at http://localhost:8000/dist/main.js
Cannot read property 'bind' of undefined
    at module.exports (http://localhost:8000/jspm_packages/npm/paperclip@1.4.10/lib/utils/bind.js:3:15)
    at new Component (http://localhost:8000/jspm_packages/npm/paperclip@1.4.10/lib/components/base.js:10:20)
    at ComponentHydrator.module.exports.protoclass.hydrate (http://localhost:8000/jspm_packages/npm/paperclip@1.4.10/lib/template/vnode/element/componentHydrator.js:20:21)
    at new View (http://localhost:8000/jspm_packages/npm/paperclip@1.4.10/lib/template/view/index.js:20:18)
    at Template.module.exports.protoclass.view (http://localhost:8000/jspm_packages/npm/paperclip@1.4.10/lib/template/index.js:76:16)
    at new Test (http://localhost:8000/dist/main.js:24:43)
    at execute (http://localhost:8000/dist/main.js:37:13)
    at m (http://localhost:8000/jspm_packages/system.js:1:6387)
    at Object.$__global.upgradeSystemLoader.e.instantiate.Promise.all.then.execute (http://localhost:8000/jspm_packages/system.js:1:8161)
    at o (http://localhost:8000/jspm_packages/es6-module-loader.js:7:18488)
```

now if i use this in page it works fine
```
<script>
    var template =paperclip.template('Generate <input type="text" class="form-control" style="width:60px;display:inline-block;" value="{{<~>numItems}}"/> items: <br/>    <repeat each="{{ range(numItems) }}" as="number">    {{~number}} <br />    </repeat>');

    // create the view from the template
    var view =  template.view({
        numItems: 1000,
        range: function (count) {
            return _.range(Math.min(count, 20000)).reverse();
        }
    });
    // render the view, and append to the DOM
 
    document.body.appendChild( view.render());
 

</script>
```

**Comment by crcn** [comment]
Hmm - seems like there might be a problem transpiling paperclip to es6. I'll do some digging around here in a bit and figure out what's going on.

**Comment by devmondo** [comment]
also even if we use this 
```
 this.template = paperclip.template($("#template").html());
```

we still get this error
```
Potentially unhandled rejection [2] SyntaxError: Error loading "main" at http://localhost:8000/dist/main.js
Expected </br> but " " found.
```

**Comment by devmondo** [comment]
i even tried to convert html to js string as below and same error so i give up :)

```
 this.template = paperclip.template(
            "Generate\n" +
            "<input type=\"text\" class=\"form-control\" style=\"width:60px;display:inline-block;\"\n" +
            "       value=\"{{<~>numItems}}\"/> items: <br/>\n" +
            "<repeat each=\"{{ range(numItems) }}\" as=\"number\"> {{~number}} <br/></repeat>"
        );
```

**Comment by devmondo** [comment]
i found one of the causes, it throws if there is empty line
so if i got 
```
hello {{ name.first }} {{ name.last }}!
--empty line here
```
it throws

**Comment by crcn** [comment]
Yeah - odd that I'm unable to reproduce any of this - just pushed a new patch. What if you try `1.4.11`?

**Comment by devmondo** [comment]
thanks a lot for responsive reply, but sadly still same error :(

maybe it has something to do with ES6? as same code works in normal script tag as i mentioned above, i use babel to transpile to ES6

**Comment by crcn** [comment]
Yep - I'll have a look at es6 stuff over the weekend.

**Comment by devmondo** [comment]
thank you very much for the effort, we cant wait for this to work :0

**Comment by crcn** [comment]
Sorry this took so long to fix - should be resolved with `1.4.16`.

**Comment by devmondo** [comment]
:+1:
